config/agents/svc/agent/agentsvc.yaml

```diff
@@ -62,10 +62,17 @@ spec:
         - mountPath: /tmp/k8s-webhook-server/serving-certs
           name: cert
           readOnly: true
+        - name: primaza-secret-volume
+          mountPath: /etc/primaza
+          readOnly: true
       serviceAccountName: primaza-svc-agent
       terminationGracePeriodSeconds: 10
       volumes:
       - name: cert
         secret:
           defaultMode: 420
           secretName: webhook-server-cert
+      - name: primaza-secret-volume
+        secret:
+          defaultMode: 420
+          secretName: primaza-secret
```

Comment:
```suggestion
          secretName: primaza-app-kubeconfig
```

Reply:
Added for agent svc
primaza-svc-kubeconfig